prow/test/integration/config/prow/cluster/webhook_server_deployment.yaml

```diff
@@ -50,6 +50,19 @@ spec:
         - name: config
           mountPath: /etc/config
           readOnly: true
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 3
+          periodSeconds: 3
+        readinessProbe:
+          httpGet:
+            path: /healthz/ready
+            port: 8081
+          initialDelaySeconds: 10
```

Comment:
The initial delays for both of these seem too high. We'll probably be ready much faster than this in practice so I think we should start checking sooner, maybe 1 and 3 seconds to start with?